### PR TITLE
Android Tip

### PR DESCRIPTION
**Pull request by jherico**
Final changes from the Android branch

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7148/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7156/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7271/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7273/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7273/HighFidelity-Beta-PR12388-458b944d2d98abb43fecae26469a11b1db8aed4c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7273/HighFidelity-Beta-PR12388-458b944d2d98abb43fecae26469a11b1db8aed4c.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7425/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7425/HighFidelity-Beta-PR12388-49dea19b269e0fa470d2c4042a0e08388ba4dd71.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7425/HighFidelity-Beta-PR12388-49dea19b269e0fa470d2c4042a0e08388ba4dd71.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7439/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7439/HighFidelity-Beta-PR12388-4e2d3f23a688c3e9420f1f5085df68024221a8c5.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7439/HighFidelity-Beta-PR12388-4e2d3f23a688c3e9420f1f5085df68024221a8c5.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7448/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7449/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7457/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7459/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7459/HighFidelity-Beta-PR12388-12d1885dffdeff7928b0fc81f26b0236876c8f9e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7459/HighFidelity-Beta-PR12388-12d1885dffdeff7928b0fc81f26b0236876c8f9e.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7507/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7507/HighFidelity-Beta-PR12388-3e552e5739058255262a2288597c1d76916e20ce.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7507/HighFidelity-Beta-PR12388-3e552e5739058255262a2288597c1d76916e20ce.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7718/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7719/app_12388_PR.apk

**Comment by jherico** [comment]
Waiting on PRs #12499 and #12519

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7720/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7720/HighFidelity-Beta-PR12388-8bfc610cfb9911950d347f09905da3a75aedcc50.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7720/HighFidelity-Beta-PR12388-8bfc610cfb9911950d347f09905da3a75aedcc50.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7760/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7760/HighFidelity-Beta-PR12388-6cc36136bdb3fa3954f05ef960cd7c08dd2cef10.dmg

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7810/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7811/app_12388_PR.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7811/HighFidelity-Beta-PR12388-3f762c790037d74b67762802ad21158eee3757d2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7811/HighFidelity-Beta-PR12388-3f762c790037d74b67762802ad21158eee3757d2.exe

File: libraries/ui/src/ui/OffscreenQmlSurface.cpp [hunk 1]
```diff
@@ -225,7 +225,7 @@ void AudioHandler::run() {
 
 void OffscreenQmlSurface::initializeEngine(QQmlEngine* engine) {
     Parent::initializeEngine(engine);
-
+    new QQmlFileSelector(engine);
```
[the remark this answers is not in the document]
Comment (jherico) [reply libraries/ui/src/ui/OffscreenQmlSurface.cpp]: no.  By passing engine as the `parent` parameter to the constructor the file selector becomes owned by the QQmlEngine and will be destroyed when it is.

**Pull request merged**